front-end/src/lib/Footer/Footer.component.tsx

```diff
@@ -0,0 +1,43 @@
+          />
+        </svg>
+      </a>
+      <a href="https://www.x.com" aria-label="www.X.com">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" fill="none">
+          <path
+            d="M64 0C28.7 0 0 28.7 0 64V384C0 419.3 28.7 448 64 448H384C419.3 448 448 419.3 448 384V64C448 28.7 419.3 0 384 0H64ZM361.1 84L257.3 202.6L379.4 364H283.8L209 266.1L123.3 364H75.8L186.8 237.1L69.7 84H167.7L235.4 173.5L313.6 84H361.1ZM323.3 335.6L153.4 110.9H125.1L296.9 335.6H323.2H323.3Z"
+          />
+        </svg>
+      </a>
+      <a href="https://www.instagram.com/" aria-label="www.Instagram.com">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" fill="none">
+          <path
+            d="M64 0C28.7 0 0 28.7 0 64V384C0 419.3 28.7 448 64 448H162.2V302.2H109.4V224H162.2V190.3C162.2 103.2 201.6 62.8 287.2 62.8C303.4 62.8 331.4 66 342.9 69.2V140C336.9 139.4 326.4 139 313.3 139C271.3 139 255.1 154.9 255.1 196.2V224H338.7L324.3 302.2H255V448H384C419.3 448 448 419.3 448 384V64C448 28.7 419.3 0 384 0H64Z"
+          />
+        </svg>
+      </a>
+    </div>
+    <p>© 2024 Mammoth Studios, All rights reserved.</p>
+  </footer>
+);
+
+export default Footer;
```

Comment:
If you do a non-default export, you'll be able to do the cool index.ts import in all your files. See above comment 😄